packages/v4/patternfly-docs/content/design-guidelines/components/menu-toggle/menu-toggle.md

```diff
@@ -6,4 +6,4 @@ subsection: menus
 
 From a design perspective, the dropdown, select, and menu toggle components are all visually the same.
 
-For more information about this component's usage, see the [dropdown](/components/dropdown/design-guidelines), [select](/components/select/design-guidelines), and [menu](/components/menu/design-guidelines) component guidelines.
+For more information about this component's usage, see the [dropdown](/components/menus/dropdown/design-guidelines), [select](/components/select/design-guidelines), and [menu](/components/menu/design-guidelines) component guidelines.
```

Comment:
The urls for the select and menu links in this sentence also need to be updated.